environment_linux.yml

```diff
@@ -16,7 +16,7 @@ dependencies:
 - bioconda::bioconductor-DEFormats
 - bioconda::bioconductor-clusterprofiler
 - conda-forge::python=3.7
-- conda-forge::scikit-learn
+- conda-forge::scikit-learn=0.23.2
```

Comment:
You may want to explicitly set the pickle version as well. It _shouldn't_ change since you have explicit versions for sk-learn and pandas, but explicitly setting it would make sure that it is correct

Reply:
Good point, i'll add this as well.